matrix_client/client.py

```diff
@@ -171,6 +175,15 @@ def send_text(self, text):
     def send_emote(self, text):
         return self.client.api.send_emote(self.room_id, text)
 
+    def send_image(self,url,name,size,mimetype,width=500,height=500):
+        extra_info = {
+            'size':size,
+            'mimetype':mimetype,
+            'width':width,
+            'height':height
+        }
+        return self.client.api.send_content(self.room_id, url,name,"image",extra_info)
```

Comment:
Good point. Since it's the same naming as the media API, this might be a better plan.